### PR TITLE
Allow additional snort options to be specified via an array property

### PR DESCRIPTION
**Pull request by chewi**
See title.

### CODE REVIEW

File: resources/service.rb [hunk 1]
```diff
@@ -57,12 +58,14 @@ def create_init
       template "/etc/systemd/system/#{svc_name}.service" do
         source 'init_systemd.erb'
         cookbook 'snort'
+        variables resource: new_resource
```
Comment (damacus) [inline resources/service.rb]: I'd prefer to explicitly pull in `new_resource.options` here so it's clearer on the other end and I don't see the benefit of pushing attribute selection into the template. I mean yes it's pretty flexible, but as a rule I tend/try to keep as much stuff out of templates.

**Comment by damacus** [comment]
I think in this case actually daemon_options, or startup_options might make the option clearer from a user point of view.

**Comment by josephholsten** [comment]
@chewi I've updated this changeset to pass the variable directly in #15. Thanks for the new feature!

**Comment by chewi** [comment]
Thanks and sorry for not coming back on this one, it slipped through the net.

**Pull request closed without merging**